assembl/static2/js/app/pages/administration.jsx

```diff
@@ -68,6 +71,10 @@ class Administration extends React.Component {
       this.putResourcesInStore(nextProps.resources);
     }
 
+    if (nextProps.sections !== this.props.sections) {
```

Comment:
not sure about this one (same for resources above), @vincentfretin as they are deep structures, the 2 arrays will always be different?

Reply:
This is an identity check here, this checks if this is the same array instance, not the content of the array.